decred/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "decred"
+version = "0.1.0"
```

Comment:
Could you set this to `0.0.1` to start.

Reply:
0.1.0 is the initial value set by Poetry, I don't know how they chose it. I don't know how you chose 0.0.1 either, :slightly_smiling_face: but sure.